### PR TITLE
Feature/store status

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu please review. Thanks

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc looks good so far. However, can you do a rebase off master and I will take a look again? Thanks.

**Comment by ezequielc** [comment]
Sure @alex-deaconu Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I rebased the branch, but the store status endpoint is not available at the moment (probably due to a branch rebuild), so the status is not working in the stage-0.

File: partials/placeholder-playlist.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
         <div ng-click="playlistItemFactory.edit(item)">
           <p class="remove-bottom playlist-item-name">{{item.name}}</p>
-          <p class="text-small text-muted remove-bottom">Status</p>
+          <p class="text-small text-muted remove-bottom" ng-class="{'text-success': ['On Trial','Subscribed'].indexOf(item.subscriptionStatus) >= 0, 'text-danger': ['Not Subscribed','Trial Expired','Cancelled','Suspended','Product Not Found','Company Not Found','Error'].indexOf(item.subscriptionStatus) >= 0}">{{item.subscriptionStatus}}</p>
```
Comment (alex-deaconu) [inline partials/placeholder-playlist.html]: @ezequielc I'm wondering if we should create a filter for this, instead of hardcoding it in the HTML. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc I tested and there's 1 use case which doesn't work:
- Show subscription status in the Playlist after adding a new Item. Once I add the item, the status is only refreshed when I switch to another placeholder and back (since status is loaded only when the placeholder is changed).
Can you take a look and fix? Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu I fixed the issue. Please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc sorry for the delay. In all honesty I feel this process is very inefficient. However, we can't do much with the current API suite we're using. Hopefully we can optimize in the future.

Otherwise, looks good :+1: .

Thanks!

**Comment by ezequielc** [comment]
Thanks Alex. I will leave this PR open while we discuss the store flow.

**Comment by ezequielc** [comment]
@alex-deaconu I rebased this branch and we are good to merge it. I will work in another branch for the store actions. Cheers.

**Comment by alex-deaconu** [comment]
@ezequielc :+1: . Thanks.

**Pull request merged**